bb-worker/Dockerfile.apt-build-python3.7

```diff
@@ -0,0 +1,24 @@
+ARG BASEOS
+FROM $BASEOS
+ARG DEBIAN_FRONTEND=noninteractive
+ARG DUMBINIT_VERSION=v1.2.2
```

Comment:
DUMBINIT_VERSION is set here but not used

Reply:
Removed!